.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: rust
+sudo: false
+
+
+script:
+ - cargo build
```

Comment:
Why do you need to explicitly define the script? I thought that `language: rust` would do that automagically?